pandas/tests/base/conftest.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas.tests.indexes.conftest import indices_dict
```

Comment:
I would rather just move the indices_dict to pandas/conftest.py and create these helper series there (as fixtures). ok to do this in 2 PRs

Reply:
I just did it in this one. Please let me know if it's what you intended.

> create these helper series there (as fixtures)

Not sure what you mean with "as fixtures". Like described in [my comment above](https://github.com/pandas-dev/pandas/pull/31701#discussion_r376716074), I don't see a better way to create this `index_or_series_obj` fixture